xil_carma_cloud/docker-compose.yml

```diff
@@ -0,0 +1,596 @@
+    volumes_from:
+      - container:carma-config:ro
+    environment:
+      - ROS_IP=172.5.0.5
+      - ROS_MASTER_URI=http://172.5.0.2:11311/
+    volumes:
+      - /opt/carma/logs/carma_2:/opt/carma/logs
+      - /opt/carma/.ros:/home/carma/.ros
+      - /opt/carma/vehicle/calibration:/opt/carma/vehicle/calibration
+      - /opt/carma/maps:/opt/carma/maps
+      - /opt/carma/routes:/opt/carma/routes3
+      - /opt/carma/yolo:/opt/carma/yolo
+    command: bash -c 'wait-for-it.sh 172.5.0.2:11311 -- rosparam load /opt/carma/vehicle/config/bridge.yml && source ~/.base-image/workspace/install/setup.bash && ros2 run ros1_bridge dynamic_bridge --multi-threads'
+
+  carma-carla-integration_2:
+    image: usdotfhwastol/carma-carla-integration:carma-system-4.5.0
+    networks:
+      carma_net_2:
+        ipv4_address: 172.5.0.6
+      carma_sim_net:
+        ipv4_address: 172.2.0.10
+    container_name: carma-carla-integration_2
+    volumes_from:
+      - container:carma-config:ro
+    environment:
+      - ROS_IP=172.5.0.6
+      - ROS_MASTER_URI=http://172.5.0.2:11311/
+    command: >
+              bash -c "export RMW_IMPLEMENTATION=rmw_fastrtps_cpp
+              && wait-for-it.sh 172.5.0.2:11311
+              && export PYTHONPATH=$PYTHONPATH:~/PythonAPI/carla/dist/carla-0.9.10-py3.7-linux-x86_64.egg:/home/carma/carla-sensor-lib/src
+              && source ~/carma_carla_ws/devel/setup.bash
+              && python3 /home/carma/carma-carla-integration/carla-health-poll/src/wait_for_carla.py
+              && roslaunch carma_carla_agent carma_carla_agent.launch 
+              role_name:='carma_2'
+              host:='172.2.0.2'
+              selected_route:='Release_test_case_2'
+              start_delay_in_seconds:='15'
+              spawn_point:='201.4,-291.5,0,0,0,-90'"
+
+  ns3_adapter_2:
+    image: usdotfhwastol/carma-ns3-adapter:carma-system-4.5.0
+    networks:
+      carma_net_2:
+        ipv4_address: 172.5.0.7
+      carma_sim_net:
+        ipv4_address: 172.2.0.11
+    container_name: ns3_adapter_2
+    volumes_from:
+      - container:carma-config:ro
+    environment:
+      - ROS_IP=172.5.0.7
+      - ROS_MASTER_URI=http://172.5.0.2:11311/
+      - NVIDIA_VISIBLE_DEVICES=all
+    volumes:
+      - /opt/carma/logs/carma_2:/opt/carma/logs
+      - /opt/carma/.ros:/home/carma/.ros
+      - /opt/carma/vehicle/calibration:/opt/carma/vehicle/calibration
+      - /opt/carma/maps:/opt/carma/maps
+      - /opt/carma/routes:/opt/carma/routes
+      - /opt/carma/yolo:/opt/carma/yolo
+      - ./ns3_adapter/adapter_2/parameters.yaml:/opt/carma/install/ns3_adapter/share/ns3_adapter/config/parameters.yaml
+    command: bash -c "wait-for-it.sh 172.5.0.2:11311 -- roslaunch ns3_adapter ns3_adapter.launch role_id:='carma_2' remap_ns:=/hardware_interface/comms"
+#############################################################################
+
+  carma-cloud:
+    image: ${DOCKER_ORG:-usdotfhwastoldev}/carma-cloud:${DOCKER_TAG:-develop}
```

Comment:
This should be the correct default values for deployment. However, I noticed that if we set DOCKER_ORG locally to test, we don't have 4.5.0 image for carma-cloud. Just documenting it.

Reply:
Yeah because it was not part of that release. The next release is 4.6.0 and will include CDASim, V2X-Hub, and CARMA-Cloud but will use the previous released images for the rest of the services.